src/FileFormats/SDPA/SDPA.jl

```diff
@@ -164,10 +164,12 @@ function Base.write(io::IO, model::Model{T}) where {T}
     end
     println(io)
 
-    index_map = Vector{Tuple{Int,Int}}(
-        undef,
-        MOI.dimension(MOI.PositiveSemidefiniteConeTriangle(max_dim)),
-    )
+    max_index_dim = if max_dim > 0
+        MOI.dimension(MOI.PositiveSemidefiniteConeTriangle(max_dim))
```

Comment:
I think this is not needed anymore, `MOI.PositiveSemidefiniteConeTriangle(0)` should work.